internal/distro/fedora/distro.go

```diff
@@ -784,6 +784,14 @@ func (t *imageType) checkOptions(customizations *blueprint.Customizations, optio
 		}
 	}
 
+	if t.name == "iot-raw-image" {
+		allowed := []string{"User", "Group", "Directories", "Files", "Services"}
+		if err := customizations.CheckAllowed(allowed...); err != nil {
+			return nil, fmt.Errorf("unsupported blueprint customizations found for image type %q: (allowed: %s)", t.name, strings.Join(allowed, ", "))
+		}
+		// TODO: consider additional checks, such as those in "edge-simplified-installer" in RHEL distros
+	}
```

Comment:
This check is run inside the block for the condition `t.bootISO && t.rpmOstree` but `iot-raw-image` isn't a boot ISO.

Reply:
It is not AFAICT... Am I missing anything? 🤔

Reply:
So the fedora seems to be OK, I think that you wanted to comment on the rhel9 change instead...